api-client/src/utils/helpers.ts

```diff
@@ -0,0 +1,13 @@
+import { Parser, ParserOptions } from 'htmlparser2'
+
+export function parseHTML(html: string, ontext: (data: string) => void, options: ParserOptions = {}): void {
```

Comment:
Can't we just return the decoded/parsed html instead of using callbacks? Would be simpler to use then imo.
We are also mainly using anonymous arrow functions so I would prefer that here as well.

Reply:
Maybe we could even set `decodeEntities` to true since we use that most of the time and it probably does not hurt in the cases we don't use it yet? But not sure about that, what do you think?

Reply:
I think that htmlparser2 expects a callback to work.

Reply:
Sure, htmplarser needs that. But you can refactor your custom `parseHTML` method to handle the usage of the callback internally, such that you just input html and get the parsed text as string as return value.